src/Sylius/Bundle/ShopBundle/templates/homepage/index.html.twig

```diff
@@ -0,0 +1,8 @@
+{% set homepage_hook = 'sylius_shop.homepage.index' %}
+{% set prefixes = [homepage_hook] %}
```

Comment:
It appears to be unused.

Reply:
It is set here because it is used in the `base` template